kyaml/yaml/rnode_test.go

```diff
@@ -406,6 +406,28 @@ func TestRNodeGetValidatedMetadata(t *testing.T) {
 				errMsg: "missing metadata.name",
 			},
 		},
+		"generateNameConfigMap": {
```

Comment:
Besides `GetValidatedMetadata` specifically, does this change impact any other functions or methods in kyaml? If so, we should have a few tests in those areas to demonstrate any behavioral changes.

Reply:
It shouldn't, in fact, I don't see too much value of `generateName` outside of its field.